--- symbolic-1.0.9/src/symbols.cc.old	2011-08-16 14:23:27.339002360 -0400
+++ symbolic-1.0.9/src/symbols.cc	2011-08-16 14:24:41.754582871 -0400
@@ -373,7 +373,7 @@
 			gripe_wrong_type_arg ("subs",args(0));
 			return retval;
 		}
-		if (!(args(1).is_list() || args(1).is_cell())) {
+		if (!(args(1).is_cell())) {
 			if (!get_symbol (args(1), the_sym)) {
 				gripe_wrong_type_arg("subs",args(1));
 				return retval;
--- symbolic-1.0.9/src/symlsolve.cc.bak	2011-08-16 14:26:15.621053366 -0400
+++ symbolic-1.0.9/src/symlsolve.cc	2011-08-16 14:26:45.954882161 -0400
@@ -46,7 +46,7 @@
 	}
 	
 	try {
-		if(args(0).is_list() || args(0).is_cell()) {
+		if(args(0).is_cell()) {
 			octave_value_list oct_eqn_list(args(0).list_value());
 			for(i=0;i<oct_eqn_list.length();i++) {
 				if(!get_relation(oct_eqn_list(i),relation)) {
@@ -67,7 +67,7 @@
 			eqns.append(relation);
 		}
 
-		if(args(1).is_list() || args(1).is_cell()) {
+		if(args(1).is_cell()) {
 			octave_value_list oct_vars(args(1).list_value());
 			for(i=0;i<oct_vars.length();i++) {
 				if(!get_symbol(oct_vars(i),expression)) {
